helm/reana/values.yaml

```diff
@@ -59,12 +59,12 @@ components:
     docs_url: https://docs.reana.io
     forum_url: https://forum.reana.io
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-ui:0.8.0
+    image: reanahub/reana-ui:0.8.1
   reana_db:
     enabled: true
   reana_server:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-server:0.8.1
+    image: reanahub/reana-server:0.8.2
```

Comment:
We need to release a new Docker image with [this change](https://github.com/reanahub/reana-server/pull/438) and modify the version here to `reana-server:0.8.3`